### ISSUE TITLE
Required validation failing

### ISSUE THREAD

**Issue by heldersepu** [swagger-api/swagger-ui#3444]
- swagger-ui version
latest 
- a swagger file reproducing the issue
http://swashbuckletest.azurewebsites.net/swagger/docs/V1

## Steps to reproduce:

1. Goto: http://swashbuckletest.azurewebsites.net/swagger/ui/index#/Default/Default_GetById
2. Click on button [Try it out]
3. Click on button [Execute]
4. The Loading will show, and console shows err:
```
Uncaught Error: Required parameter id is not provided
    at swagger-ui-bundle-js:52
    at Array.forEach (<anonymous>)
    at Object.o [as buildRequest] (swagger-ui-bundle-js:52)
    at swagger-ui-bundle-js:12
    at swagger-ui-bundle-js:6
    at swagger-ui-bundle-js:72
    at swagger-ui-bundle-js:52
    at Object.r [as executeRequest] (swagger-ui-bundle-js:61)
    at swagger-ui-bundle-js:12
    at swagger-ui-bundle-js:6
```
## Expected result
Clicking on button [Execute] when there are required param(s) should "flash & shake" the required field(s)

**Comment by owenconti** [comment]
Hi @heldersepu, I believe this is fixed in the latest version 3.0.20, which was released just last night.

Could you try updating to 3.0.20 and see if you can reproduce this issue?

**Comment by heldersepu** [comment]
Awesome! it works as intended with 3.0.20 !
